libs/form/src/next/form-control/stories/form-control.stories-arg.ts

```diff
@@ -1,12 +1,18 @@
-import { CATEGORIES, TYPES } from '@domg-wc/common-storybook';
+import { CATEGORIES, TYPES, defaultArgs, defaultArgTypes } from '@domg-wc/common-storybook';
 import { ArgTypes } from '@storybook/web-components';
-import { FormControlDefaults } from '../form-control';
+import { formControlDefaults } from '../form-control';
+import { action } from '@storybook/addon-actions';
 
-export const formControlArgs: typeof FormControlDefaults = {
-    ...FormControlDefaults,
+type FormControlArgsType = typeof defaultArgs & typeof formControlDefaults & { onVlReset: () => void };
+
+export const formControlArgs: FormControlArgsType = {
+    ...defaultArgs,
+    ...formControlDefaults,
+    onVlReset: action('vl-reset'),
 };
 
 export const formControlArgTypes: ArgTypes<typeof formControlArgs> = {
+    ...defaultArgTypes(true),
     id: {
         name: 'id',
         description: 'Het id van het veld.',
```

Comment:
Gezien we voor de juiste werking van huidige opzet altijd `name` nodig hebben voor elke form control, maar intern de `name` gaan instellen gelijk aan de `id` zou ik mss overwegen dit ook in de documentatie op te nemen; eventueel
- `id` als `required` aanduiden bij args documentatie
- bij `name` description vermelden dat we zelf intern `id` waarde gaan nemen als `name` niet staat ingesteld